opennms-container/minion/container-fs/confd/templates/org.apache.karaf.management.cfg.tmpl

```diff
@@ -0,0 +1,107 @@
+#
+# DON'T EDIT THIS FILE :: GENERATED WITH CONFD
+#
+
+#
+# The properties in this file define the configuration of Apache Karaf's JMX Management
+#
+
+#
+# Port number for RMI registry connection
+#
+rmiRegistryPort = {{getv "/karaf/management/rmi/registry/port" "1299"}}
```

Comment:
Not strictly necessary, but it may be a good idea to assign the common parts of the path to a variable and then concatenate to get the full path. Avoids churning in the future should we change key paths (not that its likely). See `confd-telemetry-feature.xml.tmpl` `$flowsPath` variable as example.

Same for other templates here.

Reply:
Sure, will do